recipe/meta.yaml

```diff
@@ -49,38 +64,30 @@ outputs:
     script: install-python.sh
     build:
       activate_in_script: True
+      string: "{{ fft_impl }}_py{{ py }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"
     requirements:
       build:
         - {{ compiler('c') }}
         - make
-        - pkg-config
+        - pkg-config >=0.18.0
         - swig >=3.0.7
       host:
+        - gsl
         - {{ pin_subpackage('lal', exact=True) }}
+        - mkl-devel {{ mkl }}  # [fft_impl == "mkl"]
+        - numpy 1.11.3  # [fft_impl == "mkl"]
+        - numpy  # [fft_impl != "mkl"]
         - python
-        - numpy
-        - six
-        - ligo-segments
-        - lscsoft-glue
-        - scipy
-        - libcblas
-        - gsl
-        - zlib
-        - fftw
-        - hdf5
       run:
+        - gsl
         - {{ pin_subpackage('lal', exact=True) }}
-        - python
-        - {{ pin_compatible('numpy') }}
-        - six
-        - python-dateutil
         - ligo-segments
         - lscsoft-glue
+        - {{ pin_compatible('numpy') }}
+        - python
+        - python-dateutil
         - scipy
-        - gsl
-        - zlib
-        - fftw
-        - hdf5
+        - six
```

Comment:
You can add `libcblas * *mkl   # [fft_impl="mkl"]` to prevent `blas=openblas and fft=mkl`